packages/web/components/PageHeader.tsx

```diff
@@ -41,63 +93,119 @@ export const PageHeader: React.FC = () => {
+          isExternal
         >
-          <title>Menu</title>
-          <path d="M0 3h20v2H0V3zm0 6h20v2H0V9zm0 6h20v2H0v-2z" />
-        </svg>
-      </Button>
+          <Image src={MetaQuests} alt="MetaQuests" />
+          Quests
+        </MenuItem>
 
-      <Box
-        display={{ base: show ? 'block' : 'none', md: 'flex' }}
-        width={{ base: 'full', md: 'auto' }}
-        alignItems="center"
-        flexGrow={1}
-        my={{ base: 8, md: 0 }}
-      >
-        <Stack
-          direction={{ base: 'column', md: 'row' }}
-          spacing={{ base: 4, md: 6, lg: 10 }}
+        <MenuItem
+          href="https://forum.metagame.wtf/c/quest/5/l/latest?board=default"
+          isExternal
         >
-          <MenuItem href="/">Players</MenuItem>
-          <MenuItem href="https://discord.gg/VYZPBnx" isExternal>
-            Discord
-          </MenuItem>
-          <MenuItem href="https://wiki.metagame.wtf/" isExternal>
-            Wiki
-          </MenuItem>
-          <MenuItem href="https://forum.metagame.wtf" isExternal>
-            Forums
-          </MenuItem>
-          <MenuItem href="https://metagame.substack.com" isExternal>
-            Blog
-          </MenuItem>
-        </Stack>
-      </Box>
+          <Image src={MetaRaids} alt="MetaRaids" />
+          Raids
+        </MenuItem>
 
-      <Box
-        display={{ base: show ? 'flex' : 'none', md: 'block' }}
-        justifyContent="center"
-        width={{ base: 'full', md: 'auto' }}
-      >
+        <Button
+          display="flex"
+          zIndex="11"
+          textDecoration="none"
+          variant="link"
+          flexDirection="column"
+          justifyContent="center"
+          alignItems="center"
+          width="calc(32rem / 5)"
+          className="filter-effect"
+          position="relative"
+          top={show ? '1rem' : '46rem'}
+          left="-0.25rem"
+          onClick={handleToggle}
+        >
+          <Image src={MetaGameLogo} alt="MetaGameLogo" height="6rem" />
+        </Button>
+
+        <MenuItem href="https://forum.metagame.wtf/" isExternal>
+          <Image src={MetaForum} alt="MetaForum" />
+          Forum
+        </MenuItem>
+
+        <MenuItem href="/">
+          <Image src={MetaPlayers} alt="MetaPlayers" />
+          Players
+        </MenuItem>
+      </Stack>
+
+      <Box justifyContent="center" width={{ base: 'full', md: 'auto' }}>
         <LoginButton />
       </Box>
+
+      <Stack
+        width="33rem"
+        direction="row"
+        flexWrap="wrap"
+        position="absolute"
+        zIndex="10"
+        top="5rem"
+        left="calc(50% - 16.5rem)"
+        padding="1rem 0"
+        transition="opacity 0.8s cubic-bezier(0.65, 0, 0.35, 1)"
+        opacity={show ? 0 : 1}
+        pointerEvents={show ? 'none' : 'inherit'}
+      >
+        <Image
+          src={MetaBox}
+          alt="MetaBox"
+          w="33rem"
+          position="absolute"
+          left="calc(50% - 16.5rem)"
```

Comment:
No magic numbers, and we shouldnt be using an image for this. Doesnt animate well, doesn't work well with responsive styles, and slow to load. As a rule of thumb, we pretty much should never be using images as UI elements with very rare exceptions.